src/main/java/duke/command/EditCommand.java

```diff
@@ -0,0 +1,54 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+/**
+ * EditCommand class handles the commands that edit a particular task in the tasklist.
+ */
+public class EditCommand extends Command {
```

Comment:
Perhaps it would be better if you split this class into DoneCommand and DeleteCommand because they themselves achieve slightly different functionalities? 👍